extensions/amp-accordion/amp-accordion.md

```diff
@@ -26,9 +26,11 @@ limitations under the License.
 
 # amp-accordion
 
-Provides a way for viewers to glance at the content outline and jump to any section. This is helpful for mobile devices where even a couple of sentences into a section requires scrolling.
-
 <table>
+  <tr>
+    <td class="col-fourty"><strong>Description</strong></td>
+    <td>A clickable stacked list of headers that collapse or expand content sections.</td>
```

Comment:
Same nit here, not high priority, but we might want to reconsider using 'clickable' since it doesn't apply to touch screens as well.

Reply:
thanks for the catch, I've updated them